test/SourceKit/Sema/educational_note_diags.swift

```diff
@@ -1,8 +1,8 @@
 extension (Int, Int) {}
 
-// RUN: %sourcekitd-test -req=sema %s -- -Xfrontend -print-educational-notes -Xfrontend -diagnostic-documentation-path -Xfrontend /educational/notes/path/prefix %s | %FileCheck %s -check-prefix=DESCRIPTIVE
+// RUN: %sourcekitd-test -req=sema %s -- %s | %FileCheck %s -check-prefix=DESCRIPTIVE
```

Comment:
Would it make sense to duplicate this test so that we check both the default behaviour and that if you explicitly provide the path then that overrides the default?